models/fivetran_log__audit_table.sql

```diff
@@ -7,13 +19,18 @@ with sync_log as (
     from {{ ref('stg_fivetran_log__log') }}
 
     where event_subtype in ('sync_start', 'sync_end', 'write_to_table_start', 'write_to_table_end', 'records_modified')
+
+    {% if is_incremental() %}
+        and created_at >= ( select max(sync_start) from {{ this }} )
+    {% endif %}
```

Comment:
Based off @DylanBaker comment [here](https://github.com/fivetran/dbt_fivetran_log/pull/29#discussion_r778713946) I would assume we should be taking a different approach here. Otherwise thee incremental strategy essentially does nothing, if I am understanding this correctly 🤷 

Based off Dylan's comment, it looks like BigQuery will do a full table scan with this logic. That will defeat the purpose and I feel we should take the other approach so to address the query cost issue.

Reply:
swapped this out! wondering if we should switch the logic in other packages with incremental models?

Reply:
also hitting some errors on circle ci but not locally?? `max_sync_start` is `None` on incremental runs and that's failing :/

Reply:
Ahhh I think this is because the table it is trying to reference doesn't exist yet. We may need to leverage the --depends-on comment for this?

Reply:
Essentially my understanding is this is being executed during the first compile, and since CircleCi is starting with a fresh environment, the `sync_start` doesn't technically exist yet. We need to explicitly tell dbt where it can find the sync_start during compile time.

Reply:
I am thinking we may need to do something like this:
https://github.com/fivetran/dbt_zendesk/blob/5922ae9df394fde85e8c3fa533ac2707bc5fa78a/models/utils/int_zendesk__calendar_spine.sql#L1-L23

Reply:
ok got this working with some various if statements...